settings.ini

```diff
@@ -3,17 +3,17 @@ autoStart=1
 initialRes=100
 
 [Resolution change]
-minRes=60
-maxRes=500
+minRes=25
+maxRes=200
 dataPullDelayMs=250
-resChangeDelayMs=1400
-minGpuTimeThreshold=0.75
+resChangeDelayMs=2000
```

Comment:
Instead of changing `resChangeDelayMs`, I would look into increasing `dataAverageSamples` **a lot** if you would be willing to test if that's any better.

Reply:
> Instead of changing `resChangeDelayMs`, I would look into increasing `dataAverageSamples` **a lot** if you would be willing to test if that's any better.

Will test this. also checked in the app with my GPU milliseconds and it returned 0.86ms in the SteamVR void, soooooooooooooo i think 1ms would be fine for most people

Reply:
Tested this, yeah, changing dataAverageSamples to 17 worked great.

Reply:
Great!